### ISSUE TITLE
Redundant 'package not installed' warning

### ISSUE THREAD

**Issue by goldingn** [zoonproject/zoon#373]
From @lizhmartin (originally [here](https://github.com/zoonproject/zoontutorials/issues/27)):

When a package is installed as a result of calling a module for which the dependent packages are not yet installed, the below warning is returned, which is redundant:

"In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE, :
there is no package called ‘biomod2’"

**Comment by goldingn** [comment]
This is an issue in `zoon::GetPackage()`. I can replicate with:

```r
remove.packages('versions')
zoon::GetPackage('versions')
```
```
Loading required package: versions
trying URL 'http://cran.rstudio.com/bin/macosx/mavericks/contrib/3.3/versions_0.3.tgz'
Content type 'application/x-gzip' length 22352 bytes (21 KB)
==================================================
downloaded 21 KB


The downloaded binary packages are in
	/var/folders/y8/bwn1d6dx6hs042dp0xz246vc0000gn/T//RtmpiNFzpA/downloaded_packages
Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called ‘versions’
```

**Comment by goldingn** [comment]
This should be fixed in #374
